client/src/components/AutoHeightVirtualList.tsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import { List, CellMeasurerCache } from "react-virtualized";
+
+interface AutoHeightVirtualListProps {
```

Comment:
Shouldn't there be a way to import and extend `ListProps` from `@types/react-virtualized`? Would clean this up and get an implementation of `rowRenderer` that doesn't use an `any`.

Reply:
Yeah, there is a way and that was my initial approach as well ([you can see it in this commit](https://github.com/TBS-EACPD/infobase/pull/1071/commits/524db7b402d5cf9bdea4121936d261e4b5b52a93))
But `ListProps` require `height`, which requires `Typeahead` to pass in `height` when passing props into `AutoHeightVirtualList`. But obviously the sole purpose of `AutoHeightVirtualList` is to control `height` so I decided to manually define type for this component

Reply:
Googled up some options, I think `interface AutoHeightVirtualListProps extends Omit<ListPRops, "height">{` should do it?

https://www.typescriptlang.org/docs/handbook/utility-types.html#omittype-keys

Reply:
Some alternate options seem to be overriding by extending with `height?: never`. Probably a use case for that patern, but the `Omit` util type seems better here.